test/jdk/javax/swing/JColorChooser/Test4887836.java

```diff
@@ -26,10 +26,12 @@
 import javax.swing.JColorChooser;
 import javax.swing.UIManager;
 
+import jtreg.SkippedException;
+
 /*
  * @test
  * @bug 4887836
- * @library /java/awt/regtesthelpers
+ * @library /java/awt/regtesthelpers /test/lib
  * @build PassFailJFrame
```

Comment:
You may explicitly mention `jtreg.SkippedException` in the `@build` tag.
```suggestion
 * @build PassFailJFrame jtreg.SkippedException
```

Reply:
Left it for now.